raw-data/EA/EA_inventories.tsv

```diff
@@ -8112,36 +8112,36 @@ InventoryID	LanguageCode	LanguageName	Segment
 2433	kjh	Khakas (Standard)	uː
 2433	kjh	Khakas (Standard)	ɔ
 2433	kjh	Khakas (Standard)	ɔː
-2434	0	Eastern Khanty (Vakh)	p
-2434	0	Eastern Khanty (Vakh)	v
-2434	0	Eastern Khanty (Vakh)	m
-2434	0	Eastern Khanty (Vakh)	t
-2434	0	Eastern Khanty (Vakh)	s
-2434	0	Eastern Khanty (Vakh)	n
-2434	0	Eastern Khanty (Vakh)	l
-2434	0	Eastern Khanty (Vakh)	ʈʂ
-2434	0	Eastern Khanty (Vakh)	ɳ
-2434	0	Eastern Khanty (Vakh)	ɭ
-2434	0	Eastern Khanty (Vakh)	r
-2434	0	Eastern Khanty (Vakh)	c
-2434	0	Eastern Khanty (Vakh)	j
-2434	0	Eastern Khanty (Vakh)	ɲ
-2434	0	Eastern Khanty (Vakh)	ʎ
-2434	0	Eastern Khanty (Vakh)	q
-2434	0	Eastern Khanty (Vakh)	ʁ
-2434	0	Eastern Khanty (Vakh)	ŋ
-2434	0	Eastern Khanty (Vakh)	iˑ
-2434	0	Eastern Khanty (Vakh)	eˑ
-2434	0	Eastern Khanty (Vakh)	æˑ
-2434	0	Eastern Khanty (Vakh)	yˑ
-2434	0	Eastern Khanty (Vakh)	øˑ
-2434	0	Eastern Khanty (Vakh)	ɯˑ
-2434	0	Eastern Khanty (Vakh)	ɑˑ
-2434	0	Eastern Khanty (Vakh)	oˑ
-2434	0	Eastern Khanty (Vakh)	ɪ
-2434	0	Eastern Khanty (Vakh)	ʏ
-2434	0	Eastern Khanty (Vakh)	ɯ̽
-2434	0	Eastern Khanty (Vakh)	ʊ
+2434	NA	Eastern Khanty (Vakh)	p
```

Comment:
agree that `NA` is better than `-` or `0`.  I'm +1 for the plan of maintaining a single mapping of InventoryID -> Glottocode -> ISOcode.  A nice feature of that would be that it could check to see whether the ISO in the raw data matched the one from the mapping doc, and issue warnings for ones that change (at least in the intial stages, that could be a useful check).